### ISSUE TITLE
Interfacing with Datasource from Soliscloud.com

### ISSUE THREAD

**Issue by CyberRagz** [deg0nz/MMM-Fronius2#6]
My Solar Plants data logger uploads data to https://www.soliscloud.com. If the API of this data source is made available is it possible to integrate it into this MM module for display data on MagicMirror.

**Comment by deg0nz** [comment]
Yes, this would be possible. But I'm limited in time and resources to implement this. So PRs are welcome :)
